src/main/java/seedu/module/logic/parser/FindModuleCommandParser.java

```diff
@@ -0,0 +1,29 @@
+package seedu.module.logic.parser;
+
+import static seedu.module.commons.core.Messages.MESSAGE_INVALID_COMMAND_FORMAT;
+
+import seedu.module.logic.commands.FindModuleCommand;
+import seedu.module.logic.parser.exceptions.ParseException;
+
+/**
+ * Parses input arguments and creates a new FindModuleCommand object
+ */
+public class FindModuleCommandParser implements Parser<FindModuleCommand> {
+    /**
+     * Parses the given {@code String} of arguments in the context of the FindModuleCommand
+     * and returns a FindModuleCommand object for execution.
```

Comment:
not sure if its better to mention the return like this or use '@return' instead. Just a small issue